amundsen_application/static/js/components/TableDetail/ExploreButton/tests/index.spec.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as ConfigUtils from 'config/config-utils';
+import ExploreButton from 'components/TableDetail/ExploreButton';
+import { TableMetadata } from 'interfaces/TableMetadata';
+import { logClick } from 'ducks/utilMethods';
+
+
+let mockExploreEnabled = true;
+let mockExploreUrl = 'https://test-website.com';
+
+jest.mock('config/config-utils', () => (
+  {
+    exploreEnabled: () => { return mockExploreEnabled },
+    generateExploreUrl: () => { return mockExploreUrl }
+  }
+));
+
+const generateExploreUrlSpy = jest.spyOn(ConfigUtils, "generateExploreUrl");
+
+describe('ExploreButton', () => {
+  const setup = (tableDataOverrides?: Partial<TableMetadata>) => {
+    const props = {
+      tableData: {
+        badges: [],
+        cluster: 'cluster',
+        columns: [],
+        database: 'database',
+        is_editable: false,
+        is_view: false,
+        key: '',
+        schema: 'schema',
+        name: 'table_name',
+        description: '',
+        table_writer: { application_url: '', description: '', id: '', name: '' },
+        partition: {
+          is_partitioned: true,
+          key: 'partition_key',
+          value: 'partition_value',
+        },
+        table_readers: [],
+        source: { source: '', source_type: '' },
+        watermarks: [],
+        ...tableDataOverrides,
+      },
+    };
+    const wrapper = shallow<ExploreButton>(<ExploreButton {...props } />)
+    return { props, wrapper };
+  };
+
+  describe('generateUrl', () => {
```

Comment:
the one test in this `describe` block should go under `render`, since we no longer have a `generateUrl` method